pkg/asset/agent/image/unconfigured_ignition.go

```diff
@@ -22,6 +23,36 @@ const (
 	unconfiguredIgnitionFilename = "unconfigured-agent.ign"
 )
 
+// GetConfigImageFiles returns the list of files or file paths to be included in the config-image.
+func GetConfigImageFiles() []string {
+	return []string{
+		"/etc/assisted/manifests/pull-secret.yaml", //nolint:gosec // not hardcoded credentials
+		"/etc/assisted/manifests/nmstateconfig.yaml",
```

Comment:
Since the nmstateconfig.yaml could be added in the config image, shouldn't the static network config files under /etc/assisted/network also be copied? And then applied by the /usr/local/bin/pre-network-manager-config.sh script?

Reply:
Good catch Richard, we had missed this since dev-scripts was using the nmstateconfig info for the unconfigured ignition. Added the optional dir to the list of files. However there seems to be a dependency issue after that, marking this as hold
/hold